test/specs/main.spec.js

```diff
@@ -40,15 +43,55 @@ describe('Main', function() {
     });
 
     describe('#init', function() {
+        var userId = 'thisisauserid',
+            appToken = 'thisisanapptoken',
+            jwt = 'thisisajwt';
```

Comment:
With and without the token? This test will only test the case where we have a token.